aspnetcore/fundamentals/index.md

```diff
@@ -151,16 +149,20 @@ ASP.NET Core supports a logging API that works with a variety of built-in and th
 
 To create logs, resolve an <xref:Microsoft.Extensions.Logging.ILogger%601> service from dependency injection (DI) and call logging methods such as <xref:Microsoft.Extensions.Logging.LoggerExtensions.LogInformation*>. For example:
 
-[!code-csharp[](index/samples_snapshot/3.x/TodoController.cs?highlight=5,13,19)]
-
-Logging methods such as `LogInformation` support any number of fields. These fields are commonly used to construct a message `string`, but some logging providers send these to a data store as separate fields. This feature makes it possible for logging providers to implement [semantic logging, also known as structured logging](https://softwareengineering.stackexchange.com/questions/312197/benefits-of-structured-logging-vs-basic-logging).
+[!code-csharp[](~/fundamentals/index/samples/6.0/RazorPagesMovie/Pages/Movies/Index.cshtml.cs?name=snippet&highlight=3-10, 16-17)]
 
 For more information, see <xref:fundamentals/logging/index>.
 
 ## Routing
 
 A *route* is a URL pattern that is mapped to a handler. The handler is typically a Razor page, an action method in an MVC controller, or a middleware. ASP.NET Core routing gives you control over the URLs used by your app.
 
+The following code, generated by the ASP.NET Core web application template, calls <xref:Microsoft.AspNetCore.Builder.EndpointRoutingApplicationBuilderExtensions.UseRouting%2A>:
+
+[!code-csharp[](~/fundamentals/startup/6.0_samples/WebAll/Program.cs?name=snippet4&highlight=17)]
+
+The `UseRouting` call can be removed because the host adds routing when it's instantiated.
+
```

Comment:
For me, it seems like showing that the templates generate the call but then saying you don't really need it can be a bit confusing. It's covered early in the *Routing basics* section of the routing docs in the "For more information" link that follow, which I think is enough.

```suggestion
```